src/Microsoft.ServiceFabric.Actors/Client/FabricActorExceptionConvertor.cs

```diff
@@ -0,0 +1,37 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.ServiceFabric.Actors.Client
+{
+    using System;
+    using Microsoft.ServiceFabric.Services.Communication;
+    using Microsoft.ServiceFabric.Services.Remoting.V2.Client;
+
+    internal class FabricActorExceptionConvertor : IExceptionConvertor
+    {
+        public bool TryConvertFromServiceException(ServiceException serviceException, out Exception actualException)
+        {
+            return this.TryConvertFromServiceException(serviceException, (Exception[])null, out actualException);
+        }
+
+        public bool TryConvertFromServiceException(ServiceException serviceException, Exception innerException, out Exception actualException)
```

Comment:
can't we have innerException default to null and avoid the above function?

Reply:
null values should come towards the end. We have out param here which should come at the end, hence cant do as you suggested.